packages/next/src/build/webpack/loaders/action-utils.ts

```diff
@@ -0,0 +1,70 @@
+const key = crypto.subtle.generateKey(
+  {
+    name: 'AES-GCM',
+    length: 256,
+  },
+  true,
+  ['encrypt', 'decrypt']
+)
+
+async function encrypt(salt: string, data: ArrayBuffer) {
+  const iv = new TextEncoder().encode(salt)
+  return crypto.subtle.encrypt(
+    {
+      name: 'AES-GCM',
+      iv,
+    },
+    await key,
+    data
+  )
+}
+
+async function decrypt(salt: string, data: ArrayBuffer) {
+  const iv = new TextEncoder().encode(salt)
+  return crypto.subtle.decrypt(
+    {
+      name: 'AES-GCM',
+      iv,
+    },
+    await key,
+    data
+  )
+}
+
+function arrayBufferToString(buffer: ArrayBuffer) {
+  let binary = ''
+  let bytes = new Uint8Array(buffer)
+  let len = bytes.byteLength
+  for (let i = 0; i < len; i++) {
+    binary += String.fromCharCode(bytes[i])
+  }
+  return binary
```

Comment:
Afaik V8 has a limit of 65535 arguments in a function. For len < 65535, you can do the following which is faster:

```js
return String.fromCharCode.apply(null, bytes);
```

Reply:
Thanks!